app/packages/core/src/Root/Root.tsx

```diff
@@ -53,16 +53,17 @@ const rootQuery = graphql`
   }
 `;
 
-export const DatasetSavedViewsQuery = graphql`
-  query RootDatasetSavedViewsQuery($name: String!) {
-    ...RootDatasetSavedViewsFragment
-  }
-`;
+// export const DatasetSavedViewsQuery = graphql`
+//   query RootDatasetSavedViewsQuery($name: String!) {
+//     ...RootDatasetSavedViewsFragment
+//   }
+// `;
 
 export const DatasetSavedViewsFragment = graphql`
   fragment RootDatasetSavedViewsFragment on Query
-  @refetchable(queryName: "EootDatasetSavedViewsQuery") {
+  @refetchable(queryName: "RootDatasetSavedViewsQuery") {
     savedViews(datasetName: $name) {
+      id
```

Comment:
nice! if you tested this and everything works as before, I am cool with this change. I'd test creating/editting/deleting saved views and makes sure the list of views gets updated properly without a hard refresh. Also please remove the commented code